frontends/web/src/locales/en/app.json

```diff
@@ -718,6 +718,7 @@
     }
   },
   "reset": {
+    "bb02Understand": "I have a backup on my SD card or have the BIP39 mnemonic written down",
```

Comment:
I would do "I have backup" only, as that covers both. Non-experts won't be confused by the cryptic `BIP39`, while experts know what's up ;)